admin/views/view_my_items.php

```diff
@@ -215,5 +232,25 @@ function loadMore(size, dataLength, item) {
 			}
 		});
 	}
+
+	window.opinoinstageHandleSelectLinks = function(elm){
```

Comment:
inline js is illegal, let's move it to file and use jquery [document ready](https://api.jquery.com/ready/)

Reply:
and looks like this function (opinoinstageHandleSelectLinks) is never used